simulation/traffic_simulator/include/traffic_simulator/entity/misc_object_entity.hpp

```diff
@@ -81,6 +81,10 @@ class MiscObjectEntity : public EntityBase
     THROW_SEMANTIC_ERROR("requestAcquirePosition function cannot not use in MiscObjectEntity");
   }
 
+  auto getDriverModel() -> const traffic_simulator_msgs::msg::DriverModel override;
```

Comment:
```
auto getDriverModel() -> const traffic_simulator_msgs::msg::DriverModel override;
```
↓
```
auto getDriverModel() const -> traffic_simulator_msgs::msg::DriverModel override;
```

Reply:
fixed in https://github.com/tier4/scenario_simulator_v2/pull/650/commits/d3a8c09835cca7d0b623b86317f54f9152eb257e